app/controllers/api/v1/onboarding_follow_recommendations_controller.rb

```diff
@@ -20,12 +20,21 @@ def onboarding_follow_recommendations
         theme_color: category['color'],
         # If any of YML accounts are invalid or not found in the database, we'll omit them from the response
         accounts: category['accounts']
-          .map { |a| { account: Account.find_remote(*username_and_domain(a['account'])), summary: a['summary'] } }
+          .map { |a| { account: find_account(a), summary: a['summary'] } }
           .reject { |a| a[:account].nil? }
       )
     end
   end
 
+  def find_account(account)
+    # TODO: this probably belongs in account_finder_concern.rb.
+    # I'm putting it here for now to avoid messing with mastodon code.
+    # if we need it elsewhere, we should move it. SD
+    username, domain = *username_and_domain(account['account'])
+    Account.find_remote(username, domain) ||
```

Comment:
`Account.find_remote` may be passed `nil` for `domain` if a handle is provided without domain (eg `@foo`). Is this OK, or should we be doing a conditional on `domain.nil?`

Reply:
`Account.find_local` works by calling `find_remote` and passing `nil` as the domain, so it's fine.  Good catch though.